spec/classes/rabbitmq_spec.rb

```diff
@@ -227,8 +234,8 @@
             is_expected.to contain_archive('rabbitmqadmin').with_source('http://1.1.1.1:15672/cli/rabbitmqadmin')
           end
 
-          it { is_expected.to contain_package('python') } if %w[RedHat Debian SUSE Archlinux].include?(facts[:os]['family'])
-          it { is_expected.to contain_package('python2') } if %w[FreeBSD OpenBSD].include?(facts[:os]['family'])
+          it { is_expected.to contain_package('python') } if %w[RedHat Debian SUSE Archlinux].include?(os_facts[:os]['family'])
+          it { is_expected.to contain_package('python38') } if %w[FreeBSD].include?(os_facts[:os]['family'])
```

Comment:
note: this may have changed by now - not sure. but at least it matches the current stuff in data, and can be adjusted in tests later if needed